packages/turf-rhumb-bearing/index.js

```diff
@@ -0,0 +1,60 @@
+// https://en.wikipedia.org/wiki/Rhumb_line
+// http://www.movable-type.co.uk/scripts/latlong.html#rhumblines
+var getCoords = require('@turf/invariant').getCoords;
```

Comment:
Yes more documentation is always better 👍 
The Gist is `getCoord` if you want to retrieve a Point Array (with point validation), `getCoords` for the entire coordinates as an Array (limited validation).